<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notepad</class>
 <widget class="QMainWindow" name="Notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Notepad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/icon/ICON/background.jpg);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="separator"/>
    <addaction name="closeAction"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="undoAction"/>
    <addaction name="redoAction"/>
    <addaction name="separator"/>
    <addaction name="pasteAction"/>
    <addaction name="copyAction"/>
    <addaction name="cutAction"/>
    <addaction name="separator"/>
    <addaction name="SelectallAction"/>
    <addaction name="separator"/>
    <addaction name="dateTimeAction"/>
   </widget>
   <widget class="QMenu" name="menuStyle">
    <property name="title">
     <string>Style(&amp;S)</string>
    </property>
    <addaction name="fontAction"/>
    <addaction name="colorAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="aboutNotepadAction"/>
    <addaction name="aboutQtAction"/>
    <addaction name="separator"/>
    <addaction name="myWebAction"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuEdit"/>
   <addaction name="menuStyle"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="undoAction"/>
   <addaction name="redoAction"/>
   <addaction name="separator"/>
   <addaction name="newAction"/>
   <addaction name="saveAction"/>
   <addaction name="openAction"/>
  </widget>
  <action name="newAction">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/ICON/new.png</normaloff>:/new/icon/ICON/new.png</iconset>
   </property>
   <property name="text">
    <string>New(&amp;N)</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/ICON/save.png</normaloff>:/new/icon/ICON/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>Close(&amp;x)</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/ICON/open.png</normaloff>:/new/icon/ICON/open.png</iconset>
   </property>
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>Sava as...(&amp;S)</string>
   </property>
  </action>
  <action name="undoAction">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/ICON/undo.png</normaloff>:/new/icon/ICON/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo(&amp;U)</string>
   </property>
  </action>
  <action name="redoAction">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/new/icon/ICON/redo.png</normaloff>:/new/icon/ICON/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo(&amp;R)</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="text">
    <string>Paste(&amp;P)</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="text">
    <string>Copy(&amp;C)</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="text">
    <string>Cut(&amp;X)</string>
   </property>
  </action>
  <action name="SelectallAction">
   <property name="text">
    <string>SelectAll(&amp;A)</string>
   </property>
  </action>
  <action name="fontAction">
   <property name="text">
    <string>Front</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="dateTimeAction">
   <property name="text">
    <string>DateTime(&amp;T)</string>
   </property>
  </action>
  <action name="aboutNotepadAction">
   <property name="text">
    <string>About Notepad</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="myWebAction">
   <property name="text">
    <string>My Web</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>undoAction</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>undo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redoAction</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>redo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pasteAction</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyAction</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cutAction</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SelectallAction</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
